docs/source/user-guide/crate-configuration.md

```diff
@@ -25,7 +25,47 @@ control DataFusion's behavior.
 
 [configuration settings]: configs.md
 
-## Add latest non published DataFusion dependency
+## Crate features
+
+The `datafusion` crate has several [features] which can be specified in your `Cargo.toml`. For example
+to enable support for `avro` you can put the following in your `Cargo.toml`:
+
+```toml
+datafusion = { version = "45", features = ["avro"] }
+```
+
+[features]: https://doc.rust-lang.org/cargo/reference/features.html
+
+You can always check the [Cargo.toml] file for the most up to date list of features.
+
+[cargo.toml]: https://github.com/apache/datafusion/blob/main/datafusion/core/Cargo.toml#L39
+
+### Default features:
+
+- `nested_expressions`: functions for working with nested type function such as `array_to_string`
```

Comment:
Would this be better in alphabetical order?

Reply:
Great idea -- done in 66541689a